docs/UserGuide.md

```diff
@@ -194,7 +194,7 @@ These terms have specific meanings in the context of FriendlyLink. For a more de
     * Open the app by entering `java -jar friendlylink.jar` into the terminal.
    
    The GUI should appear in a few seconds.
-   ![Ui](images/Ui.png)
+   ![Ui](images/readMe/Ui.png)
```

Comment:
As in removing the current `Ui.png`, and replacing it with the one you have.

Reply:
Fixed in [993815d](https://github.com/AY2223S2-CS2103T-W12-1/tp/pull/193/commits/993815d6b488787fe551c84fac284e22bc1d6cbc)